doc/source/advanced.md

```diff
@@ -34,22 +33,25 @@ under `ingress.hosts`.
 
 Note that you need to install and configure an
 [Ingress Controller](https://kubernetes.io/docs/concepts/services-networking/ingress/#ingress-controllers)
-for the ingress object to work. We recommend the
-[nginx ingress](https://github.com/kubernetes/charts/tree/master/stable/nginx-ingress)
-controller maintained by the community.
+for the ingress object to work.
 
-### Automatic HTTPS with kube-lego & Let's Encrypt
+We recommend the [nginx ingress](https://github.com/kubernetes/charts/tree/master/stable/nginx-ingress)
+controller, [**kubernetes/ingress-nginx**](https://github.com/kubernetes/ingress-nginx),
+maintained by the community. Note that Nginx maintains two additional ingress controllers.
+For most use cases, we recommend the community maintained **kubernetes/ingress-nginx**.
 
-The default automatic HTTPS support does not work if you are using an ingress
-object. You'd have to set it up yourself.
+### Ingress and Automatic HTTPS with kube-lego & Let's Encrypt
 
+When using an ingress object, the default automatic HTTPS support does not work.
+To have automatic fetch and renewal of HTTPS certificates, you must set it up
+yourself.
 
-This uses [kube-lego](https://github.com/jetstack/kube-lego) to automatically
-fetch and renew HTTPS certificates
-from [Let's Encrypt](https://letsencrypt.org/). This currently only works with
-the nginx ingress controller & google cloud's ingress controller.
+Here's a method that uses [kube-lego](https://github.com/jetstack/kube-lego)
+to automatically fetch and renew HTTPS certificates from [Let's Encrypt](https://letsencrypt.org/).
+This approach currently only works with the community maintained *nginx ingress controller*
+and *google cloud's ingress controller*.
 
-1. Make sure that DNS is properly set up (varies wildly depending on the ingress
+1. Make sure that DNS is properly set up (this may vary wildly depending on the ingress
```

Comment:
This sentence feels a bit strange to me (the "depending on the ingress controller you are using" part) given that just above we've said that you can _only_ use one of two possible ingress controllers

Reply:
ah, so *kube-lego* only works with the nginx and gclb ingress controller. So you can use any ingress controller you want, but if you want automatic HTTPs with kube-lego then you can only use these two controllers for now.

Reply:
The situation is getting better with https://github.com/jetstack/cert-manager, but I've never used that yet.